02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>from here on, I'm a bit confused</p><p>and I'm not sure if there was a problem uploading an image or if the text I can see is what I should be seeing</p><p>If this is the correct text, it would be wise to contextualize and explain it</p><p>Maybe also for an image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Replaced this with an image!</p>